atomic_reactor/plugins/flatpak_create_oci.py

```diff
@@ -28,6 +28,13 @@
 from atomic_reactor.utils.flatpak_util import FlatpakUtil
 from atomic_reactor.utils.rpm import parse_rpm_output
 
+PLATFORM_TO_ARCH_MAPPING = {
```

Comment:
I mean, there is this https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/config.py#L456.
I wanted to avoid mocking confing map in the unittest..but I guess that is the better approach, I'll update this.

Reply:
use platform_to_goarch_mapping
https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/config.py#L456

Reply:
updated